Kajabi/sage-lib
# [Sage-408] Hero - Foundations Update

## Description

Updates Hero component to Foundations design specs:
- Updates row gap to `16px`
- Updates margin for title and body to `36px`
- Updates title and body text
  - Title: Heading 4 styles
  - Body: `Grey 700`
- Adds `borderless` prop
- Adds `custom_background_color` prop

[Figma](https://www.figma.com/file/sMbtLUHSt2vfKgKjyQ3052/Sage?node-id=4760%3A22499)

### Implementation Updates
N/A

### Follow-up Items
[Fix component max width](https://kajabi.atlassian.net/browse/SAGE-450)

## Screenshots

|  Before  |  After  |
|--------|--------|
|||
<img width="771" alt="Screen Shot 2022-04-18 at 4 35 52 PM" src="https://user-images.githubusercontent.com/14791307/163881705-ddb809d3-f35e-4a24-a0a5-edd111ff9169.png">|<img width="776" alt="Screen Shot 2022-04-18 at 4 35 08 PM" src="https://user-images.githubusercontent.com/14791307/163881711-e03754cd-bce0-4590-a89a-3aa99326d861.png">



## Testing in `sage-lib`

- View [Hero](http://localhost:4000/pages/component/hero)
- Check that component matches Foundations design specs

## Testing in `kajabi-products`

(LOW) Updates Hero component to Foundations design specs.


## Related

[Sage-408](https://kajabi.atlassian.net/browse/SAGE-408)
